tests/integration/templates/cfn_if_attribute_none.yml

```diff
@@ -0,0 +1,42 @@
+Parameters:
+  Value1:
+    Type: String
+    Default: "Value1"
+  Value2:
+    Type: String
+    Default: "Value2"
+  CreateParameter:
+    Type: String
+    Default: false
+    AllowedValues:
+      - true
+      - false
```

Comment:
Just to check - these values are not booleans; does CFn translate boolean yaml values to strings if the parameter type is `String`?!